test/arelx_test_helper.rb

```diff
@@ -6,6 +6,58 @@
 
 require 'support/fake_record'
 
+$colored = ENV["TERM"] =~ /^xterm|-256color$/ ? ->(color, msg){ "\x1b[#{color}m#{msg}\x1b[89m\x1b[0m" } : ->(color, msg){ "#{msg}" }
```

Comment:
Why a global lambda instead of merely a function?

Reply:
No particular reason other than avoiding querying ENV over and  over. It's silly, so I'll convert it to a func.